test/integration/docker-compose.yml.template

```diff
@@ -454,19 +454,12 @@ services:
       --schema-sync-interval 2
     # domain should not be changed, and client should use 'domain' value as the provider, otherwise the test will fail
 
-  ganache:
-    image: trufflesuite/ganache:v7.9.2
+  hardhat:
+    image: kwildb/hardhat:latest
     ports:
       - "8545"
     networks:
       - {{ .Network }}
-    command:
-      - --wallet.hdPath
-      - m/44'/60'/0'
-      - --wallet.mnemonic
-      - test test test test test test test test test test test junk
```

Comment:
interesting the tests pass without setting the mnemonic, as we have _deployer_ that need funds, it's configured to use this wallet in `KIT_CREATOR_PK`

Reply:
Yeah, this `test test test test test test test test test test test junk` wallet is the default.  https://github.com/kwilteam/hardhat/blob/main/hardhat.config.js
We only tweak the derivation path a bit. https://hardhat.org/hardhat-network/docs/reference#accounts

Reply:
oh, where do we change the hdPath?

Reply:
In that linked hard hat config in our own docker image: https://github.com/kwilteam/hardhat/blob/main/hardhat.config.js
We can also give a cli flag to a config file, but we control the docker image so we can set the defaults right.

Reply:
ah sorry, I just totally ignored that config